src/deno-lambda.ts

```diff
@@ -146,6 +146,9 @@ export class DenoLambda extends Lambda {
 			'bin/deno': await FileFsRef.fromFsPath({
 				fsPath: join(runtimeDeno.dir, 'deno'),
 			}),
+			'deno.lock': await FileFsRef.fromFsPath({
+				fsPath: join(cwd, 'deno.lock'),
+			}),
```

Comment:
Hopefully fixes the `dump/` issue.

Reply:
Since `deno.lock` is not necessarily always present, pleased add a try/catch for this.

Reply:
Done! Anything else?